examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb

```diff
@@ -411,7 +411,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.5"
+   "version": "3.7.3"
```

Comment:
Q: Why it does not work with 3.7.5?

Reply:
It just changed when I run that notebook. Because I was using that version. It will run on both of them.

Reply:
Changed back to "3.7.5"